doc/upgrade/2.0.md

```diff
@@ -0,0 +1,36 @@
+# UPGRADE FROM 1.x to 2.0
+
+* Support for eZ Publish kernel 6.x has been dropped. Minimum supported version
+  is now 7.0
+
+* Support for PHP 5.x and 7.0 has been dropped. Minimum supported version is
+  now 7.1
+
+* All `*.class` parameters were removed from DI container. Use compiler passes
+  to set the new class or service decorators if you need to override any
+  services from legacy bridge.
+  
+* Most of the services have been marked as private in Symfony DI container.
+  You should not fetch them from container directly any more, instead, inject
+  them as constructor arguments into your services. 
+
+* HTTP 404 error conversion in legacy mode `false` siteaccesses is now turned
```

Comment:
something like this might be easier to grasp:`HTTP 404 error conversion when legacy mode is `false` in given siteaccesses is now always handled by Symfony, and config option to control this has been removed so the logic follows legacy mode setting.`